exporter/coralogixexporter/config.go

```diff
@@ -39,9 +39,8 @@ type Config struct {
 	PrivateKey string `mapstructure:"private_key"`
 
 	// Traces emitted by this OpenTelemetry exporter should be tagged
-	// in Coralogix with the following application and subsystem names
-	AppName   string `mapstructure:"application_name"`
-	SubSystem string `mapstructure:"subsystem_name"`
```

Comment:
This is effectively a breaking change and has to be done following these rules: https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#breaking-changes